specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-09-01/managedClusters.json

```diff
@@ -0,0 +1,7370 @@
+            "value": "NodePool",
+            "description": "The snapshot is a snapshot of a node pool."
+          }
+        ]
+      },
+      "description": "The type of a snapshot. The default is NodePool."
+    },
+    "ManagedClusterWorkloadAutoScalerProfile": {
+      "type": "object",
+      "description": "Workload Auto-scaler profile for the managed cluster.",
+      "properties": {
+        "keda": {
+          "$ref": "#/definitions/ManagedClusterWorkloadAutoScalerProfileKeda"
+        },
+        "verticalPodAutoscaler": {
+          "$ref": "#/definitions/ManagedClusterWorkloadAutoScalerProfileVerticalPodAutoscaler"
+        }
+      }
+    },
+    "ManagedClusterWorkloadAutoScalerProfileKeda": {
+      "type": "object",
+      "description": "KEDA (Kubernetes Event-driven Autoscaling) settings for the workload auto-scaler profile.",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable KEDA."
+        }
+      },
+      "required": [
+        "enabled"
+      ]
+    },
+    "ManagedClusterWorkloadAutoScalerProfileVerticalPodAutoscaler": {
+      "type": "object",
+      "description": "VPA (Vertical Pod Autoscaler) settings for the workload auto-scaler profile.",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable VPA. Default value is false.",
+          "default": false
+        }
+      },
+      "required": [
+        "enabled"
+      ]
+    },
+    "ManagedClusterAzureMonitorProfile": {
+      "type": "object",
+      "description": "Azure Monitor addon profiles for monitoring the managed cluster.",
+      "properties": {
+        "metrics": {
+          "$ref": "#/definitions/ManagedClusterAzureMonitorProfileMetrics"
+        }
+      }
+    },
+    "ManagedClusterAzureMonitorProfileKubeStateMetrics": {
+      "type": "object",
+      "description": "Kube State Metrics profile for the Azure Managed Prometheus addon. These optional settings are for the kube-state-metrics pod that is deployed with the addon. See aka.ms/AzureManagedPrometheus-optional-parameters for details.",
+      "properties": {
+        "metricLabelsAllowlist": {
+          "type": "string",
+          "description": "Comma-separated list of additional Kubernetes label keys that will be used in the resource's labels metric (Example: 'namespaces=[k8s-label-1,k8s-label-n,...],pods=[app],...'). By default the metric contains only resource name and namespace labels."
+        },
+        "metricAnnotationsAllowList": {
+          "type": "string",
+          "description": "Comma-separated list of Kubernetes annotation keys that will be used in the resource's labels metric (Example: 'namespaces=[kubernetes.io/team,...],pods=[kubernetes.io/team],...'). By default the metric contains only resource name and namespace labels."
+        }
+      }
+    },
+    "ManagedClusterAzureMonitorProfileMetrics": {
+      "type": "
```

Comment:
Consider model it as an enums if the item in the `verbs` has a finite set of values?
refs: https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi015-avoid-modelling-a-property-as-string-type-when-it-should-be-an-enums